src/NuGet.Clients/NuGet.VisualStudio.OnlineEnvironment.Client/NuGetClientPackage.vsct

```diff
@@ -77,8 +91,9 @@
 
     <!-- This is the guid used to group the menu commands together -->
     <GuidSymbol name="guidNuGetClientPackageCmdSet" value="{282008cc-d0db-45e1-80d1-00fabac5de92}">
-      <IDSymbol name="NuGetClientMenuGroup" value="0x1020" />
-      <IDSymbol name="CmdidRestorePackages" value="0x0100" />
+      <IDSymbol name="nuGetClientMenuGroup" value="0x1020" />
```

Comment:
nit: nuGet really bothers me. Tools VSCT mostly follows the convention of starting off with what it is, like `cmd` or `id` or `guid`.

Reply:
I'll change it to guid.

Reply:
Sorry, I find it terribly confusing to name an IDSymbol with "guid". Why guid for this one? 🤔